labs/lab6/lab6.adoc

```diff
@@ -120,11 +65,19 @@ As the root user on the OpenShift master (_master-<student_id>.rhte.sysdeseng.co
 master$ oc get nodes
 ----
 
-If successful, you should see four (3) total nodes (1 master and 2 worker nodes) with **Ready** under the _Status_ column, as opposed to (2) total nodes before (1 master and 1 worker nodes).
+If successful, you should see four (3) total nodes (1 master and 2 worker nodes) with **Ready** under the _Status_ column, as opposed to (2) total nodes before (1 master and 1 worker nodes). The role the instance plays can be seen through labels asociated with each node. The master node in this environment is also acting as a scheduldable instance that hosts platform infrastructure related components, such as the router and registry. Thus it has been set with the label `type=infra`. 
```

Comment:
Should there be more discussion about the fact we are only using 1 master and no infra nodes? That this isn't production and only for a lab?

Reply:
Could be assumed? holding off